config/routes.rb

```diff
@@ -148,7 +148,7 @@
         post "access_list/approvals", to: 'users#access_list_approvals'
       end
     else
-      resources :users, except: [:edit, :update, :new, :create], constraints: { id: /\d+/ } do
+      resources :users, except: [:new, :create], constraints: { id: /\d+/ } do
```

Comment:
I think we want to keep the `:edit, :update` here. This is the case of the `create_users` feature being turned off, which I believe is only switched off for UIC. UIC should not be able to edit/update their users because the users are managed by an external system.